src/bundle/Resources/public/js/scripts/core/popup.menu.js

```diff
@@ -35,7 +35,7 @@
                 popupMenuItem.addEventListener(
                     'click',
                     (event) => {
-                        this.popupMenuElement.classList.add(CLASS_POPUP_MENU_HIDDEN);
+                        this.handleToggle();
```

Comment:
It is necessary to call the `this.handleToggle` method because the handleToggle method toggles the class to `this.popupMenuElement` and the `this.openedMenuClass`. And when `this.openedMenuClass` parameter is passed it is necessary to toggle both elements class